src/server.h

```diff
@@ -2714,7 +2714,8 @@ void pauseActions(pause_purpose purpose, mstime_t end, uint32_t actions);
 void unpauseActions(pause_purpose purpose);
 uint32_t isPausedActions(uint32_t action_bitmask);
 uint32_t isPausedActionsWithUpdate(uint32_t action_bitmask);
-mstime_t getPausedActionTimeout(uint32_t action);
+char *getPausedPurposeString(pause_purpose purpose);
```

Comment:
I suggest removing the String from function name getPausedPurposeString(), char* already represents the return type.  I know in server.h, some function names keep the String even the return type is char*, and some do not.  Let us keep simple and normalize it.

Reply:
updated.